charts/gardener-extension-admission-gcp/charts/runtime/templates/deployment.yaml

```diff
@@ -54,6 +51,11 @@ spec:
           mountPath: /etc/gardener-extension-admission-gcp/kubeconfig
           readOnly: true
         {{- end }}
+        {{- if .Values.global.serviceAccountTokenVolumeProjection.enabled }}
+        - name: service-account-token
+          mountPath: /etc/gardener-extension-admission-gcp/serviceaccount
```

Comment:
It is not a hard requirement that the path is specific to the component. I did this in order to follow the convention used in the other `volumeMount`s but it can be changed to something like this "/etc/projected/serviceaccount". The "generic" kubeconfig will need to have some configurations in it but it can be easily generated. Values like `certificate-authority-data` and `server` will probably need to be provided anyway so I guess passing a path to the `tokenFile` will not make any big difference. Sample kubeconfig will look like this.

```yaml
apiVersion: v1
kind: Config
current-context: garden
contexts:
- context:
    cluster: garden
    user: garden
  name: garden
clusters:
- cluster:
    certificate-authority-data: <ca-data>
    server: https://kube-apiserver
  name: garden
users:
- name: garden
  user:
    tokenFile: /etc/gardener-extension-admission-gcp/serviceaccount/token
```

I am OK with both options.

Reply:
Hm, but ideally we would only have exactly one of this "generic" kubeconfig in the entire system (instead of one per component), right? This is to prevent unnecessary blowing up the etcd size. I don't think this is achievable when we encode the component name into the mount path?

Reply:
This is a valid point. I will change it to something generic.